flashinfer/jit/gemm/core.py

```
@@ -663,6 +664,38 @@ def gen_gemm_sm120_module() -> JitSpec:
+
+    nvcc_flags = current_compilation_context.get_nvcc_flags_list(
+        supported_major_versions=[12],
+    )
+
+    return gen_jit_spec(
+        "gemm_sm120_cute_mxfp8",
+        source_paths,
+        extra_cuda_cflags=nvcc_flags
+        + [
+            "-DCUTLASS_ENABLE_GDC_FOR_SM100=1",
+        ],
```

Ruff (0.15.15)

[warning] Consider `[*nvcc_flags, "-DCUTLASS_ENABLE_GDC_FOR_SM100=1"]` instead of concatenation

Replace with `[*nvcc_flags, "-DCUTLASS_ENABLE_GDC_FOR_SM100=1"]`

(RUF005)